vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/util/ExtensionUtils.java

```diff
@@ -0,0 +1,50 @@
+/*-
+ * *
+ * *
+ * Copyright (C) 2020 - 2023 the original author or authors.
+ * *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * *
+ */
+
+package org.vividus.studio.plugin.util;
+
+import java.io.File;
+import java.net.URI;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.eclipse.core.resources.IProject;
+
+public final class ExtensionUtils
+{
+    private ExtensionUtils()
+    {
+    }
+
+    public static File asFile(String uri)
+    {
+        return new File(URI.create(uri).getPath());
+    }
+
+    public static Path resolveResourcesPath(IProject project)
```

Comment:
is it used once?

Reply:
its used in CompositeStepsSaveListener and in StepDefinitionFinder, the logic there is at some point similar but I don't think it makes sense to reuse it